src/boot/efi/linux.c

```diff
@@ -162,6 +162,8 @@ EFI_STATUS linux_exec(
         if (err != EFI_SUCCESS)
                 return err;
 
+        efivar_set_time_usec(LOADER_GUID, u"LoaderTimeExecUSec", 0);
```

Comment:
hmm, I wonder if we should set StubTimeExecUSec instead?

i.e. it might be good to be able to distinguish the time spent in the boot loader from the time spent in the stub. Given that the stub nowadays does measurements and stuff it might be slow, so instead of taping over the loader timestamp I think it would be better to add a new timestamp, and then teach userspace to look at it separately.

Reply:
To me the stub is still firmly OS loader territory. And it should be fast enough to not matter.

And I am not in the mood to add the necessary pieces to userspace to display this. :joy_cat: